test/ecto/adapters/postgres_test.exs

```diff
@@ -417,31 +433,36 @@ defmodule Ecto.Adapters.PostgresTest do
   end
 
   test "interpolated values" do
```

Comment:
We don't need to change this particular test. Testing one of combinations is enough for what it aims to exercise.